webrtc.html

```diff
@@ -8736,10 +8736,11 @@ <h2>Attributes</h2>
               whether the user agent is able to transmit text asynchronously
               with script execution.) This does not include framing overhead
               incurred by the protocol, or buffering done by the operating
-              system or network hardware. If the channel is closed, this
-              attribute's value will only increase with each call to the
-              <code><a data-for="RTCDataChannel">send()</a></code> method (the
-              attribute does not reset to zero once the channel closes).</p>
+              system or network hardware. This attribute's value will only
+              increase with each call to the <code><a data-for=
+              "RTCDataChannel">send()</a></code> method as long as the <a>
+              [[\ReadyState]]</a> slot is <code>open</code>. However, the
```

Comment:
Isn't the part about the value increasing already stated above?

Reply:
I can't see it.

Reply:
I mean by "The bufferedAmount attribute must return the number of bytes of application data (UTF-8 text and binary data) that have been queued using send() but that, as of the last time the event loop started executing a task, had not yet been transmitted to the network."

Reply:
I think it's okay to be redundant here. It reasserts that the value can only increase by a call to `send` and nothing else.

But I have a question. Is the following part really necessary?

> [...] as of the last time the event loop started executing a task [...]

Isn't that always the case for pretty much everything in the world of JS (in browsers)? Or am I missing something important?

Reply:
I think it's just reasserting that. Maybe it wasn't obvious to the authors of the spec. Also, I'm fine being redundant; better than omitting information.